assets/js/components/notifications/error.js

```diff
@@ -18,6 +18,8 @@
 
 /**
  * Error Icon - Red triangle with an exclamation mark.
+ *
+ *  @return { SVGImageElement } Notification error SVG icon.
```

Comment:
Formatting consistency
```suggestion
 *  @return {SVGImageElement} Notification error SVG icon.
```